Code/Source/Spawner/ROS2SpawnerComponent.h

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+#pragma once
+
+#include <AzCore/Component/Component.h>
+#include <AzFramework/Spawnable/Spawnable.h>
+#include <AzFramework/Spawnable/SpawnableEntitiesInterface.h>
+#include <ROS2SystemComponent.h>
```

Comment:
Is the ROSSystemComponent.h include actually needed in this file?

Reply:
I changed it to including `<rclcpp/rclcpp.hpp>` and `<ROS2/ROS2Bus.h>` in `ROS2SpawnerComponent.h` and `ROS2SpawnerComponent.cpp` respectivly.